### PR TITLE
[WIP] 🎅 0.9.0

### LINKED ISSUE #646: 0.9.0

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
Fixes #646

### CODE REVIEW

**Comment by EverlastingBugstopper** [comment]
should probably merge #764 and find a fix for #696 before releasing

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,172 @@
...
+    crates.io requires tools to set a version check `User-Agent` header when requesting the latest version. Now, when `wasm-pack` performs an API request to crates.io, it sends `User-Agent: wasm-pack/0.9.0`.
+
+    [drager]: https://github.com/drager
+    [pull/658]: https://github.com/rustwasm/wasm-pack/pull/658
+    [issue/651]: https://github.com/rustwasm/wasm-pack/issues/651
+
+  - **Remove broken link from the README - [drager], [pull/635]**
+
+    [drager]: https://github.com/drager
+    [pull/635]: https://github.com/rustwasm/wasm-pack/pull/635
```
Comment (drager) [inline CHANGELOG.md]: Hmm, were these released in 0.8.1? IIRC these wasn't released and should be released in 0.9? I might be wrong though.

Comment (EverlastingBugstopper) [reply CHANGELOG.md]: oh woops! yes i added these to the wrong section - good catch

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,155 @@
 # Changelog
 
+## 🎅 0.9.0
+
+- ### ✨ Features
+
```
Comment (Pauan) [inline CHANGELOG.md]:
```suggestion

  - **Adding in `--quiet` and `--log-level` flags to control the console output - [Pauan], [pull/694]**

    The `--verbose` flag has long existed as a way to get more console output, but now there are two flags to get *less* console output:

    * `--quiet` will silence *all* stdout, so only errors will be displayed.
    * `--log-level` can be used to silence `[INFO]` or `[WARN]` output from wasm-pack.

    [Pauan]: https://github.com/Pauan
    [pull/694]: https://github.com/rustwasm/wasm-pack/pull/694
```

Comment (Pauan) [reply CHANGELOG.md]: @EverlastingBugstopper This is a last-minute PR that was merged, this adds it to the CHANGELOG.

**Comment by ashleygwilliams** [comment]
this is blocked on #770 - working on it in #771

File: CHANGELOG.md (hunk 2 above)
Comment (Pauan) [reply CHANGELOG.md]: @EverlastingBugstopper Could you please accept this change?

Comment (Pauan) [reply CHANGELOG.md]: Thanks!

**Comment by ashleygwilliams** [comment]
closing in favor of #778

**Pull request closed without merging**